projects/plugins/jetpack/_inc/lib/admin-pages/class-jetpack-redux-state-helper.php

```diff
@@ -180,6 +180,9 @@ public static function get_initial_state() {
 				'userCounts'              => Jetpack_Core_Json_Api_Endpoints::get_user_license_counts(),
 				'activationNoticeDismiss' => Licensing::instance()->get_license_activation_notice_dismiss(),
 			),
+			'hasSeenWCConnectionModal'    => Jetpack_Options::get_option( 'has_seen_wc_connection_modal', false ),
+			// Check if WooCommerce plugin is active (based on https://docs.woocommerce.com/document/create-a-plugin/).
+			'isWooCommerceActive'         => in_array( 'woocommerce/woocommerce.php', apply_filters( 'active_plugins', Jetpack::get_active_plugins() ), true ),
```

Comment:
Instead of adding this, you could rely on `isPluginActive` from `state/site/plugins` to check if the plugin is active in the Jetpack dashboard.

Reply:
To use `isPluginActive`, we'd have to do an API request when the component is mounted and that would make the page shows the main Dashboard for a quick while before redirecting the user to this new screen, so I choose to have this coming from the initial state instead.